app/models/outgoing_message_prototype.rb

```diff
@@ -163,11 +163,15 @@ def raw_message
       if @html_body.blank? && attachments.empty?
         mail.body = @plain_body
       else
-        mail.text_part = Mail::Part.new
-        mail.text_part.body = @plain_body
-        mail.html_part = Mail::Part.new
-        mail.html_part.content_type = "text/html; charset=UTF-8"
-        mail.html_part.body = @html_body
+        if !@plain_body.blank?
```

Comment:
Is there a specific reason for `#blank?`

Personally I prefer to use `#present?` 

`if @plain_body.present?`.

Sames goes for line 170.